lib/lang.ts

```diff
@@ -66,10 +69,17 @@ const getTextValue = lookupDotNotation => {
     return missingTextData;
   }
 
+  if (typeof textValue !== 'string') {
+    return missingTextData;
+  }
+
   return textValue;
-};
+}
 
-const i18n = (lookupDotNotation, options = {}) => {
+export function i18n(
+  lookupDotNotation: string,
```

Comment:
In LDL we have this typed to the [keys in the lang files](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/types/Lang.ts).  I wonder if it is worthwhile to invest the energy to repeat that over here?

Reply:
@brandenrodgers @camden11, thoughts?